bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-import.component.html

```diff
@@ -0,0 +1,45 @@
+<sm-header></sm-header>
+
+<form #form (ngSubmit)="submit()" class="tw-max-w-xl">
+  <div>
+    <label>{{ "fileUpload" | i18n }}</label>
+    <div class="file-selector">
+      <button type="button" class="btn btn-outline-primary" (click)="fileSelector.click()">
+        {{ "chooseFile" | i18n }}
+      </button>
+      {{ this.fileSelected ? this.fileSelected.name : ("noFileChosen" | i18n) }}
+    </div>
+    <input
+      #fileSelector
+      type="file"
+      id="file"
+      class="form-control-file"
+      name="file"
+      (change)="setSelectedFile($event)"
+      hidden
+    />
+    <bit-hint>{{ "acceptedFormats" | i18n }} JSON</bit-hint>
+  </div>
+  <br />
+  <div>
+    {{ "or" | i18n }}
+  </div>
+  <br />
```

Comment:
Prefer to use margin here instead of br.
```suggestion
  <div class="tw-my-3>
    {{ "or" | i18n }}
  </div>
```

Reply:
Switched to using margin